portfolio2023/index.css

```
@@ -0,0 +1,59 @@
+body {
+  background-color: rgb(178, 191, 153);
+}
+
+/* why do we include ul in our navbar class. Is it because its easier to 
+target ul like this instead of creating classes for the ul or for each 
+indiv li  */
+/* ul gives overall structure of the navbar */
+.navbar ul {
+  list-style: none;
+  border-style: ridge;
+  border-color: darkolivegreen;
+  background-color: rgb(178, 191, 153);
+  padding: 30px;
+  margin: 60px;
+  text-align: center;
+  padding-left: 40px;
+}
+
+/* targets each link of the navbar */
+.navbar li {
+  display: inline-block;
+  font-style: oblique;
+}
+
+/* text-decoration removed the underline from the text */
+.navbar a {
+  text-decoration: none;
+  padding: 60px;
+  margin: 50px 50px 50px 50px;
```

```suggestion
  margin: 50px;
```

If top, right, bottom, left are all going to be the same value, you just need to specify it once and it'll automatically apply it to all four sides.